messaging-experiments-82.json

```diff
@@ -435,7 +435,7 @@
             "frequency": {
               "lifetime": 3
             },
-            "targeting": "!homePageSettings.isDefault && homePageSettings.isCustomUrl && homePageSettings.urls[.host == 'google.com']|length > 0 && visitsCount >= 3 && userPrefs.cfrFeatures && currentDate|date - profileAgeCreated > 15770000000",
+            "targeting": "!homePageSettings.isDefault && homePageSettings.isCustomUrl && homePageSettings.urls[.host == 'google.com']|length > 0 && visitsCount >= 3 && userPrefs.cfrFeatures && currentDate|date - profileAgeCreated > 604800000",
```

Comment:
I hope this isn't too annoying.

Reply:
Good point. I guess we will find out from the dismiss&block pings in this branch.